@@ -, +, @@ 
 mdadm.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)
--- a/mdadm.c	
+++ a/mdadm.c	
@@ -691,6 +691,8 @@ int main(int argc, char *argv[])
 		"     'summaries', 'homehost', 'byteorder', 'devicesize'.\n");
 			exit(outf == stdout ? 0 : 2);
 
+		case O(INCREMENTAL,NoDegraded):
+			fprintf(stderr, Name ": --no-degraded is deprecated in Incremental mode\n");
 		case O(ASSEMBLE,NoDegraded): /* --no-degraded */
 			runstop = -1; /* --stop isn't allowed for --assemble,
 				       * so we overload slightly */
